.github/release-please.yml

```diff
@@ -1,2 +1,6 @@
 releaseType: java-yoshi
-bumpMinorPreMajor: true
\ No newline at end of file
+bumpMinorPreMajor: true
+branches:
+  - releaseType: java-lts
+    bumpMinorPreMajor: true
+    branch: java-pubsub-lts
```

Comment:
I think we should avoid calling this `lts`, instead we should do something like `1.112.0-sp`

Reply:
Sounds good to me - I created a 1.111.0-sp branch instead.  Please see https://github.com/googleapis/java-pubsub/pull/640